index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+      charset="UTF-8"
+    />
+
+    <title>Microverse Information</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="website-main">
+      <header id="header">
+        <img
+          id="header-img"
+          src="https://d92mrp7hetgfk.cloudfront.net/images/sites/misc/SmallerMicroverselogo_revised/original.jpg?1581442029"
+        />
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#information"> Information </a></li>
+            <li><a class="nav-link" href="#level-up"> Level-up</a></li>
+            <li><a class="nav-link" href="#testimonials"> Testimonials </a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <iframe
+        id="video"
+        width="420"
+        height="315"
+        src="https://www.youtube.com/embed/mY4G9ujFxZE"
+      >
+      </iframe>
```

Comment:
I think it would look better if the video had a smaller size.

Reply:
Kindly provide a reason for making each requested changes. This will the student to easily understand and implement the requested changes. :+1: